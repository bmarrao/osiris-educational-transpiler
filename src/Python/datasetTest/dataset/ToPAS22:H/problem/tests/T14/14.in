5
12 36
Zh xScis xChex EisaS aHfQo I j CaNKw kZKXn MIqhn xA GDN
aHfQo xA
xA xScis
j CaNKw
xScis I
MIqhn CaNKw
xChex MIqhn
EisaS MIqhn
aHfQo GDN
MIqhn j
kZKXn GDN
GDN xScis
I MIqhn
xChex aHfQo
xChex j
aHfQo MIqhn
j aHfQo
Zh aHfQo
I EisaS
Zh I
xChex CaNKw
Zh j
kZKXn j
aHfQo CaNKw
EisaS aHfQo
xScis aHfQo
j EisaS
xChex xA
xA EisaS
xA GDN
xScis CaNKw
xChex xScis
xScis EisaS
aHfQo I
j xA
j GDN
EisaS kZKXn
12 60
tdmn N bE HMs J G OAzwg M VQ aOgk PzASG Z
HMs aOgk
M J
tdmn HMs
aOgk PzASG
aOgk OAzwg
M bE
HMs Z
M aOgk
J OAzwg
J bE
N aOgk
tdmn OAzwg
M G
Z N
M tdmn
bE VQ
PzASG HMs
J Z
OAzwg N
Z OAzwg
VQ G
N bE
J G
PzASG M
tdmn bE
tdmn PzASG
J VQ
Z PzASG
OAzwg PzASG
tdmn VQ
PzASG J
G OAzwg
G aOgk
G N
PzASG VQ
HMs G
OAzwg M
bE OAzwg
tdmn G
J tdmn
tdmn Z
HMs J
HMs OAzwg
VQ M
Z aOgk
bE Z
HMs N
bE HMs
G bE
aOgk J
PzASG bE
aOgk VQ
tdmn aOgk
N M
bE aOgk
G Z
N tdmn
VQ OAzwg
HMs VQ
Z VQ
12 36
qPcN Kbiz PKL g ky XVqEm y k DMPgX K qdy UJppZ
g k
XVqEm qPcN
Kbiz PKL
ky Kbiz
DMPgX ky
qdy y
qPcN k
g PKL
K XVqEm
ky qPcN
qdy Kbiz
UJppZ y
qdy k
y qPcN
K g
DMPgX g
qPcN PKL
ky qdy
g qPcN
UJppZ PKL
y PKL
DMPgX k
g ky
g qdy
PKL XVqEm
DMPgX qdy
y Kbiz
PKL ky
k K
qdy qPcN
UJppZ qdy
Kbiz qPcN
ky K
k UJppZ
K y
Kbiz UJppZ
12 49
MNFb iOd HGvO HTJnr PX hDw RA L v BXf PKuQ bc
L iOd
iOd v
HTJnr bc
bc iOd
HTJnr L
HTJnr RA
PKuQ v
HTJnr MNFb
BXf L
v hDw
MNFb RA
hDw PKuQ
bc PX
iOd MNFb
MNFb PX
PKuQ bc
hDw bc
iOd PKuQ
BXf iOd
HTJnr iOd
HGvO MNFb
HGvO L
L v
PX v
PX L
PX HTJnr
hDw RA
HGvO PX
MNFb BXf
MNFb v
RA PX
HTJnr BXf
hDw PX
L MNFb
PKuQ L
HGvO HTJnr
iOd PX
bc MNFb
RA v
RA PKuQ
PKuQ MNFb
hDw BXf
PKuQ BXf
RA BXf
MNFb hDw
BXf PX
L RA
HGvO BXf
HGvO iOd
12 66
pyI YpiR jfRQ TjLW l S RyEYf hbII Mc EF VhHy D
hbII pyI
EF YpiR
EF VhHy
YpiR l
TjLW l
EF Mc
pyI jfRQ
RyEYf Mc
VhHy YpiR
l pyI
pyI TjLW
RyEYf S
EF jfRQ
S D
D Mc
YpiR RyEYf
pyI RyEYf
Mc jfRQ
EF RyEYf
VhHy hbII
TjLW RyEYf
l jfRQ
hbII EF
jfRQ hbII
S hbII
l RyEYf
VhHy TjLW
VhHy Mc
VhHy jfRQ
pyI EF
RyEYf D
S jfRQ
YpiR pyI
YpiR S
l S
EF TjLW
hbII D
TjLW D
YpiR TjLW
pyI VhHy
EF l
pyI S
hbII TjLW
VhHy l
S TjLW
Mc l
RyEYf VhHy
l D
Mc TjLW
hbII Mc
YpiR hbII
TjLW jfRQ
YpiR jfRQ
l hbII
EF S
hbII RyEYf
Mc YpiR
Mc pyI
VhHy S
jfRQ RyEYf
D VhHy
EF D
jfRQ D
YpiR D
D pyI
S Mc
