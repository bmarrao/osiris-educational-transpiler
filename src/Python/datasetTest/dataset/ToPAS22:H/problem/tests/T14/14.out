Possivel: 1152000
Impossivel
Possivel: 3271680
Impossivel
Impossivel
